doc/user/user/dashboard/index.md

```diff
@@ -124,7 +124,7 @@ You will be shown a confirmation and your browser will reload the current page.
 User Settings
 -------------
 
-The settings menu allows you to view and edit the details of your own user account. This menu also allows you to update your email subscription preferences.  You can find the settings menu in the top, right-hand corner of the page, if you have a [Gravatar](https://en.gravatar.com/) account associated with your email in IRIDA, your gravatar will be displayed:
+The settings menu allows you to view and edit the details of your own user account. This menu also allows you to update your email subscription preferences.  You can find the settings menu in the top, right-hand corner of the page under the user icon:
```

Comment:
If the style is changing this drastically, pictures in the docs that focus on the user icon should be redone.  At first look I can find this one: https://irida.corefacility.ca/documentation/user/user/dashboard/#user-settings

Reply:
Isn't the one you referenced this one that I already replaced?  As I stated in my previous reply, this is just a baby step to replacing the full menu bar, so all the images will have to be replaced at that time.

Reply:
The only one I would definitely like changed is the image immediately below this comment.  Its directly focusing on the icon which now looks different.  Whatever icon we choose should be in this screenshot please.

Reply:
I think the correct image has been updated with the solid icon.